exotica_python/src/pyexotica/interactive_rho_tuning.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import print_function
+import os
+from time import time
+import Tkinter as tk
+import rospkg as rp
+
+__all__=['Tuning']
+
+class Tuning(object):
+
+    def __init__(self, problem):
+
+        # Initial setup
+        self.master = tk.Tk()
+        self.problem = problem
+
+        # Set title
+        self.master.winfo_toplevel().title("Rho Tuning")
+
+        # Set icon
+        # note, rp.RosPack().get_path('exotica') throws error
```

Comment:
Yep, because the core package is now `exotica_core` and `exotica` is only a metapackage - which does not work with ROS pack. A metapackage is like a stack, i.e., you can use `rosstack`, cf. https://github.com/ros/rospack/issues/26. So your solution is:

```python
rp.RosStack().get_path('exotica')
```